### PR TITLE
Only show "needs approval" PRs in "Needs Attention" when they have LGTM.

### PR DESCRIPTION
**Pull request by rmmh**
Also, don't count the pending Tide status as a pending test.

### CODE REVIEW

**Comment by krzyzacy** [comment]
if a PR does not have LGTM but need me to review, isn't the PR still need my attention?

**Comment by rmmh** [comment]
"Needs Approval" is not the same as needing a review. In kubernetes/kubernetes, top-level code owners don't want to look at changes for approval until they've already gone through review rounds.

File: gubernator/view_pr.py [hunk 1]
```diff
@@ -183,19 +183,33 @@ def serial(obj):
...
+            self.response.write(json.dumps(prs, sort_keys=True, default=serial, indent=True))
         elif fmt == 'html':
             if user:
                 user = InsensitiveString(user)
                 def acked(p):
-                    if 'lgtm' in p.payload.get('labels', {}):
-                        return True  # LGTM is an implicit Ack
+                    labels = p.payload.get('labels', {})
+                    if 'lgtm' in labels:
+                        if not (user in p.payload.get('approvers', [])
```
Comment (stevekuznetsov) [inline gubernator/view_pr.py]: Why both `not (item in collection)` and `item not in collection` in one `if`?

Comment (stevekuznetsov) [reply gubernator/view_pr.py]: Should this logic be DRYed out and named in a function of `labels` and `approvers`?

Comment (rmmh) [reply gubernator/view_pr.py]: Fixed. It's not used more than once, so it stays as one line for now.

Comment (stevekuznetsov) [reply gubernator/view_pr.py]: Same logic that you added to `gubernator/filters.py` in this PR

**Comment by rmmh** [comment]
@stevekuznetsov PTAL

File: gubernator/view_pr.py (hunk 1 above)
Comment (rmmh) [reply gubernator/view_pr.py]: Done.

**Pull request merged**